operator/pkg/helm/fs_renderer.go

```diff
@@ -33,7 +33,7 @@ type FileTemplateRenderer struct {
 // NewFileTemplateRenderer creates a TemplateRenderer with the given parameters and returns a pointer to it.
 // helmChartDirPath must be an absolute file path to the root of the helm charts.
 func NewFileTemplateRenderer(helmChartDirPath, componentName, namespace string) *FileTemplateRenderer {
-	scope.Infof("NewFileTemplateRenderer with helmChart=%s, componentName=%s", helmChartDirPath, componentName)
+	scope.Debugf("NewFileTemplateRenderer with helmChart=%s, componentName=%s", helmChartDirPath, componentName)
```

Comment:
i'd leave this as info or remove if it's not really useful. debug scope is going to be reserved for voluminous dumps like objects.